File: library/src/main/java/com/hansoolabs/and/view/RecyclerViewScroller.java
package com.hansoolabs.and.view;

import android.support.v7.widget.RecyclerView;

/**
 * Set RecyclerView, Scroll it
 * Created by brownsoo on 2017. 8. 23..
 */

public interface RecyclerViewScroller {
    void setRecyclerView(RecyclerView recyclerView);
    RecyclerView.OnScrollListener getOnScrollListener();
    void scrollTo(float scrollProgress, boolean fromTouch);
}

File: quick/src/main/kotlin/com/hansoolabs/and/view/RecyclerViewScroller.kt
package com.hansoolabs.and.view

import androidx.recyclerview.widget.RecyclerView

/**
 * Set RecyclerView, Scroll it
 * Created by brownsoo on 2017. 8. 23..
 */

interface RecyclerViewScroller {
    val onScrollListener: RecyclerView.OnScrollListener
    fun setRecyclerView(recyclerView: RecyclerView)
    fun scrollTo(scrollProgress: Float, fromTouch: Boolean)
}
